bigbluebutton-html5/imports/startup/server/logger.js

```diff
@@ -1,15 +1,30 @@
 import { Meteor } from 'meteor/meteor';
 import Winston from 'winston';
 
-let Logger = new Winston.Logger();
-
-// Write logs to console
-Logger.add(Winston.transports.Console, {
-  prettyPrint: false,
-  humanReadableUnhandledException: true,
-  colorize: true,
-  handleExceptions: true,
-});
+let Logger = new (Winston.Logger)({
```

Comment:
Same goes with the colors, its not explicitly needed.. and its only purpose right now is to change the default color of the debug messages because they are set to a dark blue which just seemed hard to see on the console. They can be omitted as well.